openquake/risklib/asset.py

```diff
@@ -860,14 +860,16 @@ def _read_csv(self):
             with open(fname, encoding='utf-8') as f:
                 fields = next(csv.reader(f))
                 header = set(fields)
+                missing = expected_header - header - {'exposure', 'country'}
```

Comment:
Is 'country' a reserved name? Then we should document it.

Reply:
Yes, you should document it in the manual, Anirudh! ;-)